src/main/java/com/github/manolo8/darkbot/Bot.java

```diff
@@ -29,10 +29,10 @@ public static void main(String[] args) throws IOException {
             e.printStackTrace();
         }
 
-        String path = Bot.class.getProtectionDomain().getCodeSource().getLocation().getPath();
-        if (System.console() == null && (path.endsWith(".jar") || path.endsWith(".exe"))) {
+        //String path = Bot.class.getProtectionDomain().getCodeSource().getLocation().getPath();
+        //if (System.console() == null && (path.endsWith(".jar") || path.endsWith(".exe"))) {
```

Comment:
same as before, cleanup after yourself, don't leave commented code around if it's not a temporal thing